libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -1194,20 +1196,17 @@ void WaveClip::ApplyStretchRatio(
    auto newSequences = GetEmptySequenceCopies();
    bool swappedOnce = false;
 
-   Finally Do { [&, trimLeftBeforeStretch = GetTrimLeft(),
-                 trimRightBeforeStretch = GetTrimRight(),
-                 offsetBeforeStretch = GetSequenceStartTime()] {
-      if (success)
-      {
-         this->mClipStretchRatio = targetRatio;
-         this->mRawAudioTempo = this->mProjectTempo;
-         assert(GetStretchRatio() == targetRatio);
```

Comment:
Ignore what I said about exception safety:  this is still right.

But I would like to conserve this assertion of the postcondition.